### ISSUE TITLE
[Debian] Missing spl and spl-dkms 0.6.5.1, breaing zfs-dkms 0.6.5.1 building and making initrd unbootable

### ISSUE THREAD

**Issue by amitie10g** [zfsonlinux/pkg-zfs#170]
I recently updated the ZFS-related packages to 0.6.5 without issues. But when I updated to 0.6.5.2, no spl and spl-dkms 0.6.5.2 are found in the packages available (just the 0.6.5), making the dkms process unable to find a proper source of spl that matches the zfs sources, and therefore breaking the whole zfs-dkms building.

This desynchronization of spl and zfs dkms packages ocurred before, so I report that.

Also, an important issue is the dissapearing of previous versions of ZFS-related packages, but the SPL-related packages still have previous versions. Therefore, is possible to keep the previous versions of ZFS-related packages?

The contents of my sources.list.d/zfsonlinux.list:

```
## This file is installed by the zfsonlinux package.

deb [arch=amd64] http://archive.zfsonlinux.org/debian wheezy main
#deb-src [arch=amd64] http://archive.zfsonlinux.org/debian wheezy main

deb http://archive.zfsonlinux.org/debian wheezy-daily main
#deb-src [arch=amd64] http://archive.zfsonlinux.org wheezy-daily main

deb [arch=amd64] http://archive.zfsonlinux.org/debian jessie main
#deb-src [arch=amd64] http://archive.zfsonlinux.org/debian jessie main

deb [arch=amd64] http://archive.zfsonlinux.org/debian jessie-daily main
#deb-src [arch=amd64] http://archive.zfsonlinux.org/debian jessie-daily main
```

Aptitude:
![image](https://cloud.githubusercontent.com/assets/2096562/9982737/faebe784-5fbb-11e5-9a73-724c949f9115.png)
(0.6.5-1 != 0.6.5.1)

And no other sources.list conflicting with this list. I re-imported the repo signature and just updated the packages.

**Comment by amitie10g** [comment]
Well, I uninstalled an reinstalled everything (except the libraries and GRUB, that I upgraded between the other packages.

dkms reported no errors when building zfs-dkms and spl-dkms, but when I rebooted the system, apparently it hangs without any prompt. Then, after several minutes, I pressed CONTROL+D, and the system continues to boot, but I noticied some slowdown compared with the version 0.6.5.

I'll try with Dracut to see what happens.
